### PR TITLE
Right Click Menu: add feature flag

### LINKED ISSUE #6153: Right Click Menu Feature Flag Implemented
Hide Right Click Menus behind feature flag for development.

Acceptance criteria:

Include feature flag name in comments
Menus should be hidden behind flag.

### PR DESCRIPTION
**Pull request by samwhale**
## Context

Right Click Menus in the editor

## Summary

Adds a feature flag.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6153

### CODE REVIEW

File: includes/Experiments.php [hunk 1]
```diff
@@ -219,6 +219,17 @@ public function get_experiment_groups() {
...
 		return [
+			/**
+			 * Author: @samwhale
+			 * Issue: 6153
+			 * Creation date: 2021-06-07
+			 */
+			[
+				'name'        => 'enableRightClickMenus',
+				'label'       => __( 'Enable Right Click Menus', 'web-stories' ),
+				'description' => __( 'Adds a contextual shortcut menu when right clicking in the editor.', 'web-stories' ),
```
Comment (BrittanyIRL) [inline includes/Experiments.php:230]: just mentioning this here so we can keep it in the back of our brains for this feature, we need to make sure "right click" on a keyboard also works! (Shift + F10)

**Review by BrittanyIRL — APPROVED** [review]

**Review by swissspidy — APPROVED** [review]

File: includes/Experiments.php (hunk 1 above)
Comment (samwhale) [reply includes/Experiments.php:230]: GOOD CALL

Comment (samwhale) [reply includes/Experiments.php:230]: @BrittanyIRL said in a chat offline that we won't be opening the right click menu with the keyboard. The functionality of the shortcuts are bound to keys already.

Comment (BrittanyIRL) [reply includes/Experiments.php:230]: @swissspidy we can circle back to this, but it currently seems like all right click menu functionality is available in the keyboard shortcuts.

**Pull request merged**